apps/frontend/src/components/SideBar/SideBarRight.tsx

```diff
@@ -25,12 +28,22 @@ const SideBarRight: React.FunctionComponent<SideBarRightProps> = (props) => {
     const isComponentProp = (prop: object): prop is ComponentProp => {
         return !notRenderedPropertyTypes.includes(typeof prop);
     };
+
     const dispatch = useDispatch();
 
-    const handleChangeProperty = (id: number, newVal: string, propName: string) => {
-        // emitter.emit('componentPropertyChanged', { id: 's', propName, value: newVal });
-        dispatch(updateComponent({ id, propName, value: newVal }));
+    const handleChangeProperty = (id: number, newValue: string, propName: string) => {
+        dispatch(updateComponent({ id, propName, value: newValue }));
+        dispatch(
+            updateActiveComponentProps({
```

Comment:
I really don't understand why you do that

Reply:
Because some Property components are inter-dependant.
The possible values for ones can depend on the values of others.
That's why each change should be saved in order to be taken into account by the Property components if needed.

It is the case for example for the column number and the grid template, for the ColumnLayout component.

Reply:
Ok dude